src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -14,9 +14,6 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- ToDo: Remove this P2P reference once new packages are produced and updated -->
-    <ProjectReference Include="..\..\System.Net.Requests\ref\System.Net.Requests.csproj">
-    </ProjectReference>
-    <!-- ToDo: Remove this P2P reference once new packages are produced and updated -->
     <ProjectReference Include="..\..\System.CodeDom\ref\System.CodeDom.csproj" />
```

Comment:
We don't, but I didn't change it as I think we want to remove the CodeDom dependency in System.Xml. I will remove this anyway with the rest of changes on this PR.